### DISCUSSION TITLE
A Proposal for Chinese Documentation on fabric.js 6

### DISCUSSION THREAD

**Discussion by Nicox-gogo** [fabricjs/fabric.js#9715] [General]
Hi,

I am a developer from China and I am thrilled to hear about the upcoming release of fabric.js 6. Currently, there are many developers in China who are either learning or using fabric.js, and they are eagerly anticipating the new version! However, due to language barriers, some developers find it difficult to read English documentation. Therefore, I had an idea that perhaps the community could work together to create a Chinese documentation.

Alternatively, if you could direct me to any existing documentation related to fabric.js 6, I would be happy to translate them step by step. This would also allow me to improve my own understanding of the framework.

Thank you all for your valuable contributions.

Best regards.

**Comment by asturur** [comment]
On top of that english documentation is outdated.
For the website, after various attempts and failures i **THINK** i settled on astro + starlight + astrowind.
The theme i picked up support translations at least for some parts, if not we will find a solution that works
